### PR TITLE
Add jdownloader2 cask

### PR DESCRIPTION
**Pull request by jonathanwiesel**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
jDownloader [is already present](https://github.com/caskroom/homebrew-cask/blob/cc10e32b5d978e8305b11984c01bbdb89628597f/Casks/jdownloader.rb). If this version is newer, the current one should be removed from here and added to [caskroom/versions](https://github.com/caskroom/homebrew-versions) (with the version in the name), while the one you’re submitting should replace it here.

File: Casks/jdownloader2.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask :v1 => 'jdownloader2' do
+  version :latest
+  sha256 :no_check
+
+  if MacOS.release >= '10.7'
```
Comment (vitorgalvao) [inline Casks/jdownloader2.rb]: This is reversed from our preferences, and should use the release symbol. i.e.:

```ruby
if MacOS.release <= :snow_leopard
  url 'http://installer.jdownloader.org/JD2Setup_10_6_or_lower.dmg'
else
  url 'http://installer.jdownloader.org/JD2Setup.dmg'
end
```

**Comment by jonathanwiesel** [comment]
Oops, my bad, jDownloader2 is already on [caskroom/versions](https://github.com/caskroom/homebrew-versions/blob/master/Casks/jdownloader2.rb)

**Comment by tsparber** [comment]
For those searching for jd2 and landing here:
https://github.com/winkelsdorf/homebrew-custom/blob/master/Casks/jdownloader2.rb

**Pull request closed without merging**